docs/building-on-etherlink/transactions.md

```diff
@@ -273,6 +273,45 @@ run();
 
 ```
 
+## Getting instant confirmations
+
+Beginning with EVM node 0.48 and the version 6.0 upgrade, Etherlink supports instant confirmations.
+You can send a transaction with the `eth_sendRawTransactionSync` method and receive an instant confirmation from the sequencer that it intends to put the transaction in the next block, plus basic information about the completed transaction, such as its gas cost.
+
+Sending the transaction with the `eth_sendRawTransactionSync` method is the same as the `eth_sendRawTransaction` method: you sign the transaction and include it in the `data` parameter in the same way plus the optional `pending` value, as in this example:
+
+```bash
+curl --request POST \
+     --url https://node.shadownet.etherlink.com \
+     --header 'accept: application/json' \
+     --header 'content-type: application/json' \
+     --data '
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "params": [
+    "0x88a747dbc7f84e8416dc4be31ddef0",
+    "pending"
+  ],
+  "method": "eth_sendRawTransactionSync"
+}
+'
+```
+
+If you pass `latest` instead of `pending`, the sequencer waits until the transaction is in a block to send the confirmation.
+Etherlink supports this `pending` value only on the `eth_sendRawTransactionSync` method, not on any other methods.
+
+The sequencer returns information about the transaction, such as its gas price and gas cost.
```

Comment:
Sam here I think could be interesting to mention we return a receipt.
Side note, returning this format and using this name for the endpoint isn't out of my mind we follow a draft EIP on ethereum : https://eips.ethereum.org/EIPS/eip-7966 (but we added the optional block parameter to fill our needs). Might be interesting to mention

Reply:
Makes sense. How's this? https://github.com/etherlinkcom/docs/pull/393/commits/d29fae408be145bdc738e63514b2639f7a48e4d9

> When the sequencer executes the transaction and intends to put it in the next block, the sequencer returns a receipt for the transaction that includes information such as its gas price and gas cost. This receipt matches the specification for the [`eth_getTransactionReceipt`](https://ethereum.org/developers/docs/apis/json-rpc/#eth_gettransactionreceipt) endpoint except that the `blockHash` field is always `0x000...` because the block has not been created yet. The `blockNumber` field provides the correct level for the block that the transaction will be in. You can take this response as a confirmation that the sequencer will put the transaction in the next block.